firefox-ios/Client/Frontend/Library/Bookmarks/Edit Bookmark/EditBookmarkViewController.swift

```diff
@@ -41,9 +41,23 @@ class EditBookmarkViewController: UIViewController,
                                                     size: CGSize(width: 0, height: UX.bookmarkCellTopPadding)))
         view.tableHeaderView = headerSpacerView
     }
+
+    private lazy var saveBarButton: UIBarButtonItem =  {
+        let button = UIBarButtonItem(
+            title: String.Bookmarks.Menu.EditBookmarkSave,
+            style: .done,
+            target: self,
+            action: #selector(saveButtonAction)
+        )
+        return button
+    }()
+
     var onViewWillDisappear: (() -> Void)?
     var onViewWillAppear: (() -> Void)?
     private let viewModel: EditBookmarkViewModel
+    private var isRootViewController: Bool {
+        return navigationController?.viewControllers.first == self
+    }
```

Comment:
We don't ever want to create folders when we enter from anywhere other than the LibraryPanel?

Reply:
If so, might want to add a comment here that it's intentional behavior and add it to the ticket, I feel like QA might file a bug otherwise.

Reply:
Good call - discussed with UX and decided this action should always be present.